src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/General.BrowseObject.xaml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--Copyright, Microsoft Corporation, All rights reserved.-->
+<Rule
+  Name="ConfigurationGeneralBrowseObject"
+  DisplayName="General"
+  PageTemplate="generic"
+  Description="General"
+  OverrideMode= "Replace"
```

Comment:
I just kept them all together so that if we eventually need to take ownership of this file we can simply just delete the copy in MSBuild.